src/Thumbnails/Thumbnails.test.js

```diff
@@ -2,19 +2,23 @@ import React from 'react';
 import { render, screen } from '@testing-library/react';
 import '@testing-library/jest-dom';
 import Thumbnails from './Thumbnails';
-import { Router, MemoryRouter } from 'react-router';
-import userEvent from '@testing-library/user-event';
+import { MemoryRouter } from 'react-router';
+import { springResident } from '../mockResidentData';
```

Comment:
Nice import of springResident from mockData!

Reply:
💐